### PR TITLE
General info and CNCF Stories

### PR DESCRIPTION
**Pull request by archyufa**
Updates
* Organizers update
* Slack, Twitter, Youtube info
* #1 CNCF Stories event video and slides

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
 * [Quebec](quebec/README.md)
-  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), CNCF Ambassador
   * Sebastien Thomas [@prune998](https://github.com/prune998) SRE, [Tetrate](https://www.tetrate.io/)
+  * Carol Tardif, [Ticketmaster](https://www.ticketmaster.ca/)
 * [Toronto](toronto/README.md)
-  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), CNCF Ambassador
-* [Waterloo-Kitchener](waterloo-kitchener/README.md)
-  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), CNCF Ambassador  
-  * Chris Volik [psycotica0](https://github.com/psycotica0), Production engineer, Shopify
+  * Jason Hadi [jasonhadi](https://twitter.com/jasonhadi), SRE, [Ecobee](https://www.ecobee.com/en-us/)
```
Comment (jasonhadi) [inline README.md]:
```suggestion
  * Jason Hadi [@jasonhadi](https://github.com/jasonhadi), SRE, [ecobee](https://www.ecobee.com/)
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
+  * Dima Kassab, [@dima_kassab](https://twitter.com/dima_kassab), Customer Engineer, [Google Cloud](https://cloud.google.com/) 
+* [Waterloo-Kitchener](waterloo-kitchener/README.md)  
+  * Chris Volik [psycotica0](https://github.com/psycotica0), Production engineer, [Shopify](https://www.shopify.ca/)
+
+
+## Join our Community!
+
+### Slack
+
+Join our vibrant K8s Canada Slack community to ask you questions, collaborate bettween other Canadian cities, meet people with same interest and learn about latest CNCF, Kubecon, KCDs news - [http://K8scanadaslack.herokuapp.com](http://K8scanadaslack.herokuapp.com)
```
Comment (jasonhadi) [inline README.md]:
```suggestion
Join our vibrant K8s Canada Slack community to ask your questions, collaborate between other Canadian cities, meet people with same interests, and learn about the latest CNCF, Kubecon, and KCDs news - [http://K8scanadaslack.herokuapp.com](http://K8scanadaslack.herokuapp.com)
```

**Review by jasonhadi — COMMENTED** [review]
Minor changes!

File: README.md [hunk 3]
```diff
@@ -1,19 +1,47 @@
...
 Github repository with all Meetup information and slides from Cloud Native Eastern Canada Meetups:
-
+* [Online Canadian Cloud Native Stories](stories/README.md)
+* [Online Canadian Tech Talks](techtalks/README.md)
 * [Montreal](montreal/README.md)
   * Julia Simons Director of Marketing, [CloudOps](cloudops.com)
-  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), CNCF Ambassador
+  * Ayrat Khayretdinov [@archyufa](https://github.com/archyufa), [CNCF Ambassador](https://www.cncf.io/people/ambassadors/)
   * Anthony Dahane [@anthonydahanne](https://github.com/anthonydahanne), Software engineer, [Adaptive](https://weareadaptive.com/)
+  * Alex Gervais [@alex_gervais](https://twitter.com/alex_gervais), Senior Software Engineer, [Datawire](Datawire.io)
```
Comment (alexgervais) [inline README.md]:
```suggestion
  * Alex Gervais [@alex_gervais](https://twitter.com/alex_gervais), Senior Software Engineer, [Datawire](https://www.datawire.io/)
```

**Pull request merged**